geps/gep-1713/index.md

```diff
@@ -115,6 +115,16 @@ type ListenerSetSpec struct {
 	// 2. ListenerSet ordered by creation time (oldest first)
 	// 3. ListenerSet ordered alphabetically by “{namespace}/{name}”.
 	//
+	// Regarding Conflict Management, Listeners in a ListenerSet follow the same
+	// rules of Listeners on a Gateway resource.
+	// 
+	// This validation should happen within all of the ListenerSets attached to a 
```

Comment:
```suggestion
	// This validation should also happen within all of the ListenerSets attached to a 
```
This might disambiguate when the listeners should be validated within the listenerset context and in the gateway context